gqlgen.go

```diff
@@ -128,7 +131,9 @@ func (a Tracer) InterceptField(ctx context.Context, next graphql.Resolver) (inte
 // requests.  The service parameter should describe the name of the
 // (virtual) server handling the request. extension parameter may be empty string.
 func Middleware(opts ...Option) Tracer {
-	cfg := config{}
+	cfg := config{
+		RequestVariablesBuilder: RequestVariables,
```

Comment:
Can we move it outside config initialization, to be consistent with the current code? For example, we can implement it a few rows below like TracerProvider